### DISCUSSION TITLE
Robocup Junior Queensland Australia

### DISCUSSION THREAD

**Discussion by DamoM73** [pybricks/support#1636] [General]
I am not sure if this the correct forum, but I am looking for a chance for Pybricks to demonstrate at both the Queensland and Australian Robocup Junior Championships this year.

I'm a secondary school teacher using Lego Robotics is both classroom and extra-curricular activities. The new Australian curriculum now requires students to be writing code using a "general purpose programming language" (ie. no more Scratch). I initially tried using Python via the Spike App, but found it's API inconsistent and poorly documented, and it's Bluetooth is flakey at best. 

Somehow I got onto Pybricks, and it has been awesome. I now use it in all classes, and in my extra-curricular robotics program. The latter is the motivation behind this message.

I am a member of the [Robocup Junior Australian](https://www.robocupjunior.org.au/) Queensland Committee.  Robocup Junior is an international competition for school aged competitors. Although it is open any robotics platform, the vast majority of competitors use Lego kits. 

At our State and National competitions we have industry displays, and this year both events are being held in Brisbane Australia. I was wondering if there would be any interest in having a Pybricks display at these events. Given the reliability of the platform and teh recent release of block based code, there may be many schools interested in exploring using Pybricks.

The Queensland State Championships are on the 17th and 18th of August and the National Championships are from the 18th to the 20th of October.

**Comment by laurensvalk** [comment]
> I initially tried using Python via the Spike App, but found it's API inconsistent and poorly documented, and it's Bluetooth is flakey at best.
> 
> Somehow I got onto Pybricks, and it has been awesome. I now use it in all classes, and in my extra-curricular robotics program. The latter is the motivation behind this message.

Great to hear that Pybricks has worked well for you and your students. Thank you for sharing your experience!

> I was wondering if there would be any interest in having a Pybricks display at these events. 

I don't know if visiting will be possible but I'd be happy to stay in touch just in case. We would be happy to provide digital materials for posters or other materials if you like. I'll reach out to the Mastodon profile listed on your GitHub profile. Thanks!
